autosubmit/autosubmit.py

```diff
@@ -5323,7 +5327,7 @@ def set_status(expid, noplot, save, final, filter_list, filter_chunks, filter_st
                                     current_length) + " jobs.")
                         else:
                             Log.info(job_list.print_with_status(
-                                statusChange=performed_changes))
+                                status_change=performed_changes))
```

Comment:
Writing tests to cover the `Monitor.generate_output_txt` function, I had to write code for this line, and confirmed the bug #2330 . Fixed it here, and the test case added should prevent future regressions :+1: :bug: :hiking_boot: